functions/api/image/nfts/asset/nftImage.test.ts

```diff
@@ -17,5 +17,5 @@ const invalidAssetImageUrl = [
 
 test.each(invalidAssetImageUrl)('invalidAssetImageUrl', async (url) => {
   const response = await fetch(new Request(url))
-  expect(response.status).toBe(404)
+  expect([404, 500]).toContain(response.status)
```

Comment:
one of these is a flake, and one of these is the actual expectation.
this will make it less flaky, but at the expense of not actually running the test.

could you retry if it is a 500?